src/main/java/build/buildfarm/cas/DigestMismatchException.java

```diff
@@ -19,15 +19,11 @@
 import java.io.IOException;
 
 public class DigestMismatchException extends IOException {
```

Comment:
I'd rather make accessors for these than not retain them.

Reply:
fixed